lib/module.php

```diff
@@ -629,3 +635,90 @@ function solusiovps_restart(array $params)
         return $e->getMessage();
     }
 }
+
+function solusiovps_ListAccounts(array $params)
+{
+    try {
+        $statusMap = [
+            "0" => WHMCS\Service\Status::ACTIVE,
+            "1" => WHMCS\Service\Status::SUSPENDED,
+        ];
+        $accounts = [];
+
+        $serverId = (int)$params['serverid'];
+        $serverParams = Server::getParams($serverId);
+        $serverResource = new ServerResource(Connector::create($serverParams));
+
+        $current_page = 1;
+        do {
+            $serversOnPage = $serverResource->listServerPage($current_page);
+            $lastPage = Arr::get($serversOnPage, 'meta', [])['last_page'];
+
+            foreach (Arr::get($serversOnPage, 'data', []) as $server) {
+                $accounts[] = [
+                    'email' => $server['user']['email'],
+                    'username' => $server['user']['email'],
+                    'domain' => $server['name'],
+                    'uniqueIdentifier' => $server['name'],
+                    'product' => $server['plan']['name'],
+                    'primaryip' => $server['ip_addresses']['ipv4'][0]['ip'],
+                    'created' => Carbon::parse($server['created_at'])->format('Y-m-d H:i:s'),
+                    'status' => $statusMap[$server['is_suspended']],
+                ];
+            }
+            $current_page++;
+        } while ($current_page <= $lastPage);
+
+        return [
+            'success' => true,
+            'accounts' => $accounts,
+        ];
+    } catch (Exception $e) {
+        return [
+            'success'  => false,
+            'error' => $e->getMessage(),
+        ];
+    }
+}
+
+function solusiovps_syncAccount(array $params)
+{
+    try {
+        if (!empty(SolusServer::getByServiceId($params['serviceid']))) {
+            return [
+                'success' => "Account is already synced",
+            ];
+        }
+
+        $serverId = (int)$params['serverid'];
```

Comment:
You can just put this variable in place, it used just once, no reason to extract it to an additional variable

`$serverParams = Server::getParams((int)$params['serverid']);`